content/practices/iteration-pre-planning/index.md

```diff
@@ -0,0 +1,84 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: false
+
+title: "Iteration Pre-Planning"
+linkTitle: "Iteration Pre-Planning"
+description: "An optional planning session to make the Iteration Planning Meeting (IPM) more productive if needed."
+tags: ["Delivery"]
+length: "30 min - 1 hour"
+participants: "Product managers, 1-2 engineers"
+image: "default-cover.png"
+lastmod: "2021-07-15"
+date: "2021-07-14"
+why: 
+- If a team finds that an excessive amount of time in the [Iteration Planning Meeting (IPM)](/practices/ipm) is spent on revising stories before estimation, editing or grooming them in a smaller group beforehand can help the larger meeting produce better results.
```

Comment:
Please remove this link as well due to #1072, which actually applies to all front-matter / metadata.

Reply:
This seems unrelated to #1072. Unlike the `description` section, the link in the `why` section renders correctly, but its route is resolved differently than the link in the main body of the article. Commit `9467191c85e9bcd13791fc3ccfa3f59e5042f2d6` should fix it.